test/pummel/test-crypto-dh.js

```diff
@@ -12,23 +12,25 @@ assert.throws(
   function() {
     crypto.getDiffieHellman('unknown-group');
   },
-  /^Unknown group$/,
+  /^Error: Unknown group$/,
   'crypto.getDiffieHellman(\'unknown-group\') ' +
   'failed to throw the expected error.'
 );
 assert.throws(
   function() {
     crypto.getDiffieHellman('modp1').setPrivateKey('');
   },
-  /^crypto\.getDiffieHellman\(\.\.\.\)\.setPrivateKey is not a function$/,
+  /^TypeError: crypto\.getDiffieHellman\(\.\.\.\)\.setPrivateKey/ +
```

Comment:
I don't think this does what you're attempting to do. You'll probably have to use the `RegExp()` constructor here and build the regex out of strings.